### DISCUSSION TITLE
Lancer en 1 clic une liste d'enrichissements

### DISCUSSION THREAD

**Discussion by camilledesalabert** [Inist-CNRS/lodex#2019] [Ideas]
L'idée serait de pouvoir lancer en 1 action unique une longue liste d'enrichissements (cas d'usage : import d'un modèle sur un nouveau corpus et besoin de relancer tous les enrichissements sans qu'il soit nécessaire de les modifier)

**Comment by touv** [comment]
https://github.com/Inist-CNRS/lodex/issues/2186
